src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -153,15 +154,20 @@ await _context.Entry(methodology.ScheduledWithRelease)
         {
             return await _persistenceHelper
                 .CheckEntityExists<Methodology>(id, HydrateMethodologyForManageMethodologySummaryViewModel)
-                .OnSuccess(methodology => methodology.Status != request.Status
-                    ? UpdateMethodologyStatus(methodology, request)
-                    : UpdateMethodologyDetails(methodology, request))
+                .OnSuccess(methodology => UpdateMethodologyStatus(methodology, request))
+                .OnSuccess(methodology => UpdateMethodologyDetails(methodology, request))
                 .OnSuccess(_ => GetSummary(id));
         }
 
         private Task<Either<ActionResult, Methodology>> UpdateMethodologyStatus(Methodology methodologyToUpdate,
             MethodologyUpdateRequest request)
         {
+            if (!request.IsStatusUpdateForMethodology(methodologyToUpdate))
+            {
+                // Status unchanged
+                return Task.FromResult(new Either<ActionResult, Methodology>(methodologyToUpdate));
```

Comment:
If we make this method async, couldn't we remove the `Task.FromResult`? 

Think we would need to await the `CheckCanUpdateMethodologyStatus` to avoid the compiler giving us a warning about not awaiting anything.

Reply:
Ah yep, I've done this now 🙂